### PR TITLE
Rename properties in LoginViewController

### PR DESCRIPTION
**Pull request by willsb**
Some of the names were just bad.

Squash whenever

### CODE REVIEW

File: Toggl.Daneel/ViewControllers/LoginViewController.cs [hunk 1]
```diff
@@ -13,8 +13,8 @@ namespace Toggl.Daneel.ViewControllers
 {
     [MvxChildPresentation]
     public partial class LoginViewController : MvxViewController<LoginViewModel>
-    {
-        private const int ForgotPasswordLabelOffset = 27;
+    {
+        private const int ForgotPasswordLabelOffset = 27;
```
Comment (amulware) [inline Toggl.Daneel/ViewControllers/LoginViewController.cs]: Lot's of whitespace changes for some reason?

File: Toggl.Daneel/ViewControllers/LoginViewController.cs [hunk 2]
```diff
@@ -103,44 +103,46 @@ public override void ViewDidLoad()
...
-            Password.AttributedPlaceholder = 
-                new NSAttributedString(Resources.LoginSignUpPasswordPlaceholder, stringAttributes);
-
-            ForgotPassword.SetTitle(Resources.LoginForgotPassword, UIControlState.Normal);
+        {
+            var stringAttributes = new CTStringAttributes(
+                new UIStringAttributes { ForegroundColor = UIColor.White.ColorWithAlpha(0.5f) }.Dictionary
+            );
+
+            EmailTextField.TintColor = UIColor.White;
```
Comment (amulware) [inline Toggl.Daneel/ViewControllers/LoginViewController.cs]: This appears to be doing more than rename things.

File: Toggl.Daneel/ViewControllers/LoginViewController.cs (hunk 1 above)
Comment (willsb) [inline Toggl.Daneel/ViewControllers/LoginViewController.cs]: CRLF shenanigans

File: Toggl.Daneel/ViewControllers/LoginViewController.cs (hunk 2 above)
Comment (willsb) [inline Toggl.Daneel/ViewControllers/LoginViewController.cs]: The caret color was wrong, so I changed it. 🙈 
Can be done in another PR, if so you request

**Pull request merged**